src/components/event-alert-send/EventAlertSendCurrentStatus.tsx

```diff
@@ -0,0 +1,85 @@
+import { useTranslations } from 'next-intl';
+import type { ReactNode } from 'react';
+
+import Details from '@/ui/details/details';
+import { Table, TableBody, TableCell, TableCellHeader, TableHead, TableRow } from '@/ui/table/Table';
+import Tag from '@/ui/tag/tag';
+import { capitalized } from '@/utils/Helpers';
+
+import type { EventAlertData, EventAlertType } from './EventAlertTypes';
+
+type EventAlertSendCurrentStatusProps = {
+  type: EventAlertType;
+  detailsSummary: string;
+  detailsContent: ReactNode;
+  data: EventAlertData;
+};
+
+const EventAlertSendCurrentStatus = ({ type, detailsSummary, detailsContent, data }: EventAlertSendCurrentStatusProps) => {
+  const t = useTranslations('Forms.Send_alert.Distibution_status');
```

Comment:
```suggestion
  const t = useTranslations('Forms.Send_alert.Distribution_status');
```
I can recommend you `Code Spell Checker` VS extensions to avoid this cases

Reply:
nice i will install